test/helpers/vscodefake/memento.ts

```diff
@@ -12,6 +12,9 @@ export class TestMemento implements vscode.Memento {
         }
         return value;
     }
+    keys(): readonly string[] {
```

Comment:
I'm not sure what the changes in this file are for.

Reply:
API change in newer vscode (interface defines also 'keys').
Without this change tests wont compile.